verilog/analysis/checkers/always_ff_non_blocking_rule.h

```diff
@@ -48,7 +50,26 @@ class AlwaysFFNonBlockingRule : public verible::SyntaxTreeLintRule {
   // Diagnostic message.
   static const char kMessage[];
 
+  // Collected violations.
   std::set<verible::LintViolation> violations_;
+
+  //- Configuration ---------------------
+  bool catch_modifying_assignments_ = false;
+  bool waive_for_locals_ = false;
+
+  //- Processing State ------------------
+  // Inside an always_ff block
+  int inside_ = 0;
+
+  // Stack of inner begin-end scopes
+  using Scope =
```

Comment:
If you make this a struct
```c++
struct Scope { int syntax_tree_depth; int inherited_locals; };
```

Then it becomes self-explanatory and also there is no wondering what `.first` or `.second` is.

Reply:
(interesting, I wrote this remark not on the final commit, but then github didn't show it in the review overview so I thought it was lost ... so this is why this rough same comment appears twice. Same with the documenting parameters)